### PR TITLE
Fix/react errors

### PR DESCRIPTION
**Pull request by aminlatifi**

### CODE REVIEW

File: packages/website/src/features/Packages/Function.tsx [hunk 1]
```diff
@@ -113,10 +114,10 @@ export const Function: FC<{
...
-          const provider = new ethers.providers.JsonRpcProvider(
-            publicClient.chain.rpcUrls.public.http[0] as string
-          );
-          await handleTxnError(cannonOutputs, provider, e);
+          // const provider = new ethers.providers.JsonRpcProvider(
+          //   publicClient?.chain?.rpcUrls?.public?.http[0] as string
+          // );
+          // await handleTxnError(cannonOutputs, provider, e);
           console.error(e);
         } catch (e2) {
```
Comment (MohammadPCh) [inline packages/website/src/features/Packages/Function.tsx:122]: I think we don't need the `try catch` here anymore.

File: packages/website/src/features/Packages/FunctionInput/NumberInput.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export const NumberInput: FC<{
       step="1"
       v-model="value"
       borderColor={isInvalid ? 'red.500' : 'whiteAlpha.400'}
-      is-invalid={isInvalid}
+      is-invalid={isInvalid.toString()}
```
Comment (MohammadPCh) [inline packages/website/src/features/Packages/FunctionInput/NumberInput.tsx:28]:
```suggestion
      isInvalid={isInvalid.toString()}
```

File: packages/website/src/features/Packages/FunctionInput/AddressInput.tsx [hunk 3]
```diff
@@ -18,11 +18,11 @@ export const AddressInput: FC<{
       type="text"
       bg="black"
       borderColor={isInvalid ? 'red.500' : 'whiteAlpha.400'}
-      is-invalid={isInvalid}
+      is-invalid={isInvalid.toString()}
```
Comment (MohammadPCh) [inline packages/website/src/features/Packages/FunctionInput/AddressInput.tsx:21]:
```suggestion
      isInvalid={isInvalid.toString()}
```

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
@aminlatifi Thanks,
Just added some tiny suggestions.

File: packages/website/src/features/Packages/Function.tsx (hunk 1 above)
Comment (aminlatifi) [reply packages/website/src/features/Packages/Function.tsx:122]: I know but I kept it to fix the issue with handleTxError. For some reason we cannot use that function in the react package

**Comment by aminlatifi** [comment]
@MohammadPCh Thanks, please see my updates

**Review by MohammadPCh — APPROVED** [review]
@aminlatifi LGTM, Thanks.
Feel free to merge it when you want.

**Review by noahlitvin — APPROVED** [review]

**Pull request merged**